pattern-1/bosh-release/README.md

```diff
@@ -1,242 +1,96 @@
-# WSO2 API Manager BOSH Release
+# BOSH release for WSO2 API Manager deployment <br>pattern 1
 
-A BOSH release for deploying WSO2 API Manager 2.1.0 with Analytics on BOSH Director. 
+This repository includes a BOSH release that can be used to deploy WSO2 API Manager 2.1.0 deployment pattern 1
+configured to use a MySQL database on BOSH Director.
 
-## Quick Start Guide
+## Create and deploy the BOSH Release in BOSH Lite
```

Comment:
As a convention it might be better to keep this title as Quick Start Guide. After the title a description can be added to explain that BOSH Lite is used for this purpose and for production deployments refer a different section of the README.md. May be we can call that Installation Guide.